README.md

```diff
@@ -270,6 +274,36 @@ checkpointing so the job restarts near where it was interrupted.
     --cluster xpk-test --filter-by-job=$USER
     ```
 
+
+## GPU usage
+
+In order to use XPK for GPU, you can do so by using `device-type` flag.
+
+*   Cluster Create (provision reserved capacity):
+
+    ```shell
+    # Find your reservations
+    gcloud compute reservations list --project=$PROJECT_ID
+    # Run cluster create with reservation.
+    python3 xpk.py cluster create \
+    --cluster xpk-test --device-type=h100-80gb-8 \
+    --num-slices=2 \
+    --reservation=$RESERVATION_ID
+    ```
+
+*   Run a workload:
+
+    ```shell
+    # Find your reservations
```

Comment:
Missing the commands for running a workload

Reply:
Can you delete the extra cluster create command here and keep just the workload  command in the "Run a workload" section